projects/observability/src/shared/graphql/request/handlers/entities/query/entity/entity-graphql-query-handler.service.ts

```diff
@@ -57,8 +57,8 @@ export class EntityGraphQlQueryHandlerService implements GraphQlQueryHandler<Gra
       timeRange: request.timeRange,
       properties: request.properties,
       filters: [new GraphQlEntityFilter(request.id, request.entityType)],
-      // If querying for a single API, then always want to includeInactive
-      includeInactive: request.entityType === ObservabilityEntityType.Api
+      // If querying for a single API by ID, then always want to includeInactive
```

Comment:
@skjindal93 does this work for non-api entities? Just wondering if there was a reason it was limited before.

Reply:
It does in my testing, buy maybe @skjindal93 can confirm. I was the one that put the original limitation in just because I was only working with APIs. It may have not been needed even then.

Reply:
Yes. `includeInactive` works entity type agnostic. Can be used with any of the entity types